aiidalab_widgets_base/utils/loaders.py

```diff
@@ -0,0 +1,38 @@
+from importlib.resources import Package, files
+
+from IPython.display import Javascript, display
+
+
+def load_css_stylesheet(package: Package, filename=""):
```

Comment:
We currently don't check types so I'd remove it. Or alternatively, type the other parameter and return parameter as well.

Reply:
The other parameter is typed via its default (str). The return type is None, so not strictly required. Do you feel differently about this? I'm not against it. In principle, I know packages such as pydantic enforce typing even in cases where static analysis tools can assert it.

Reply:
I see. I've added the explicit `str` type :)